crates/loader/src/local.rs

```diff
@@ -416,7 +416,12 @@ impl LocalLoader {
                 continue;
             }
 
-            let app_root_path = src.strip_prefix(&self.app_root)?;
+            let app_root_path = match src.strip_prefix(&self.app_root) {
```

Comment:
A more concise way to express this in current versions of Rust is to use a let-pattern-match with an `else` clause, e.g.

```rust
let Ok(app_root_path) = src.strip_prefix(&self.app_root) else {
    bail!(...);
}
```